scripts/check-for-coingecko-id.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
I realize this is already written so feel free to say no but I think it would be better for this check to be with the other unit tests. I think it would have been just a new lines of JS to check this in `test/unit/chains.test.ts`. WDYT?

Reply:
yup, you are absolutely right. originally had more in the script before toning down to just this check, but yeah it should be in the tests - will move it!